Dockerfile

```diff
@@ -22,48 +21,30 @@ RUN apt-get update \
     libxml2-dev \
     libxslt-dev \
     zlib1g-dev \
-    python-software-properties \
     ghostscript \
     python-celery \
     python-sphinx \
-    openjdk-9-jre-headless \
-    python-virtualenv \
+    openjdk-8-jre \
     locales \
     pkg-config \
     gcc \
     libtool \
     automake
 
-RUN  locale-gen en_US.UTF-8
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
 
 RUN useradd -m onadata
-RUN mkdir -p /srv/onadata && chown -R onadata:onadata /srv/onadata
+RUN mkdir -p /srv/onadata && chown -R onadata:onadata /srv
 USER onadata
-RUN mkdir -p /srv/onadata/requirements
-
-ADD requirements /srv/onadata/requirements/
 
+COPY . /srv/onadata
 WORKDIR /srv/onadata
 
-RUN virtualenv /srv/onadata/.virtualenv
-RUN . /srv/onadata/.virtualenv/bin/activate; \
-    pip install pip --upgrade && pip install -r requirements/base.pip
-
-ADD . /srv/onadata/
+RUN python3.6 -m venv /srv/.virtualenv \
+  && /srv/.virtualenv/bin/pip install pip --upgrade  \
+  && /srv/.virtualenv/bin/pip install -r requirements/base.pip
 
 ENV DJANGO_SETTINGS_MODULE onadata.settings.docker
 
-USER root
-
-# for local development tmux is a nice to have
-RUN apt-get install -y tmux
-
-RUN rm -rf /var/lib/apt/lists/* \
-  && find . -name '*.pyc' -type f -delete
-
-USER onadata
-
-# Run tmux to use bash shell.
-RUN echo "set-option -g default-shell /bin/bash" > ~/.tmux.conf
```

Comment:
Sad to see this go, but maybe I will find a way to run arbitrary commands in the `docker-entry.sh` file to setup a dev environment. As stated, I use this mainly for development purposes, which means, I run tests within the container via `docker exec -it onadata_web_1 bash`. Sometimes this maybe also running management commands to do migrations as well as the Django shell.

Reply:
As such, Tmux is a key part of my workflow.